src/tests/contract_test/wallet_test.cljs

```diff
@@ -84,3 +87,32 @@
      (h/logout)
      (rf-test/wait-for
        [::logout/logout-method])))))
+
+(defn get-main-account
+  [accounts]
+  (:address (first accounts)))
+
+(def test-password integration-constants/password)
+
+(defn assert-derived-account
+  [response]
+  (is (= (:address response) (:address response)))
+  (is (= (:public-key response) (:public-key response)))
+  (is (= "m/43'/60'/1581'/0'/0" (:path (first response)))))
+
+(deftest wallet-create-derived-addresses
```

Comment:
better the test name matches the rpc event.
Can you follow the format I did in the other tests?
e.g wallet-get-derived-addressess-contract

Actually I think we need to adjust the other tests names too - we can do separately though

Reply:
Good idea, I will change it